src/data/biomes/badlands.ts

```diff
@@ -1,12 +1,29 @@
 import { Biome } from "#app/data/biome";
-import { townPokemonPool, townTrainerPool, townWeatherPool, townTerrainPool } from "#app/data/biomes/town";
+import { townPokemonPool, townTrainerPool, townTerrainPool } from "#app/data/biomes/town";
 import { BiomeId } from "#enums/biome-id";
+import { WeatherType } from "#enums/weather-type";
 
+/**
+ * 2/15 of sandstorm, 1/3 of sun, 8/15 of none during dawn/day
+ * 20% of sandstorm otherwise
```

Comment:
NIT: Keep the denominators the same for ease of reading:
```suggestion
 * 2/15 of sandstorm, 5/15 of sun, 8/15 of none during dawn/day
 * 20% of sandstorm otherwise
```